package.json

```diff
@@ -11,14 +11,15 @@
   ],
   "main": "riemann",
   "engines": {
-    "node": "> 0.6.0"
+    "node": ">= 0.12"
```

Comment:
Is it a good idea to require this level of support? Seems like at a min we should be supporting LTS, we should think of the customers of this library.

Reply:
My reasoning was that we have to depend on the lowest possible version our dependencies depend on.
In this case 'node-protobuf' requires >=0.12.
Please correct me if I'm wrong.

Reply:
Oh did it? OK ha.